examples/mobile/ios/Configuration/InternalReleaseSettings.xcconfig

```diff
@@ -0,0 +1,9 @@
+PRODUCT_BUNDLE_IDENTIFIER = io.dolby.videocall-internal
```

Comment:
for Android, it is recommended to add a suffix starting with a '.'

Shouldn't we do `io.dolby.videocall.internal` for internal release?

Reply:
Hi @ldanilina - It's a common thing in iOS to use `-` in bundle identifiers than using `.` and commonly used format is `com.{orgname}.{appname}`. 

The naming you suggested will also work, but I suggest to keep it as is - because the profiles are already created in the developer portal. More over its a test build and we don't miss anything if the identifiers are not in parity.